test/powershell/engine/Help/HelpSystem.Tests.ps1

```diff
@@ -31,7 +31,7 @@ function RunTestCase
 
     $moduleName = "Microsoft.PowerShell.Core"
 
-    UpdateHelpFromLocalContentPath $moduleName $tag
+    UpdateHelpFromLocalContentPath $moduleName $tag -Scope 'AllUsers'
```

Comment:
Does this need to be `AllUsers`?  Seems like most of the help tests can validate against CurrentUser and we should have a few tests explicitly validating `AllUsers`

Reply:
Fixed.